AmplifyPlugins/Core/AWSPluginsCoreTests/Model/GraphQLRequest/GraphQLRequestNonModelTests.swift

```diff
@@ -72,3 +72,139 @@ class GraphQLRequestNonModelTests: XCTestCase {
         XCTAssertEqual(input["id"] as? String, todo.id)
     }
 }
+
+class GraphQLRequestJSONNonModelTests: XCTestCase {
```

Comment:
NonModel  was renamed as Embeddable types, can we take this opportunity to keep it consistently as Embeddable? move class to separate file like `class GraphQLRequestEmbeddableTypeTests_JSON` with file name `GraphQLRequestEmbeddaleTypeTests+JSON.swift`?

Reply:
and rename `GraphQLRequestNonModelTests` to `GraphQLRequestEmbeddableTypeTests` (depending on what u decide from above comment)